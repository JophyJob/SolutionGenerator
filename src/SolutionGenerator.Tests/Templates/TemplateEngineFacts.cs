// --------------------------------------------------------------------------------------------------------------------
// <copyright file="TemplateEngineFacts.cs" company="WildGums">
//   Copyright (c) 2012 - 2016 WildGums. All rights reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------

namespace SolutionGenerator.Tests.Templates
{
    public class TemplateEngineFacts
    {
        
    }
}