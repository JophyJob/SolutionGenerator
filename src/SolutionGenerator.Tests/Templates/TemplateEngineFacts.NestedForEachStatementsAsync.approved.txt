// --------------------------------------------------------------------------------------------------------------------
// <copyright file="Models.cs" company="My company">
//   Copyright (c) 2008 - 2018 My company. All rights reserved.
// </copyright>
// --------------------------------------------------------------------------------------------------------------------


namespace SomeSolution.Models
{
    
    public class Record_1
    {
        
        public string Target 1 { get; set; }
        
        public string Target 2 { get; set; }
        
        public string Target 3 { get; set; }
        
        public string Target 4 { get; set; }
        
        public string Target 6 { get; set; }
        
        public string Target 8 { get; set; }
        
        public string Target 10 { get; set; }
        
    }
    
    public class Record_2
    {
        
        public string Target 1 { get; set; }
        
        public string Target 2 { get; set; }
        
        public string Target 3 { get; set; }
        
        public string Target 4 { get; set; }
        
    }
    
}